src/test/java/integration/BaseTest.java

```diff
@@ -58,7 +58,7 @@ public static CompletableFuture<Entry> createEntry(RepositoryApiClient client, S
 
     public static CompletableFuture<Boolean> allFalse(List<TemplateFieldInfo> arr) {
         for (TemplateFieldInfo templateFieldInfo : arr) {
-            if (templateFieldInfo.isRequired) {
+            if (templateFieldInfo.isIsRequired()) {
```

Comment:
the generated method name is not appropriate. maybe we can check that if a boolean property starts with is, then don't add is to the getter method name.

Reply:
Indeed, I feel this too. I think Swagger codegen's way of doing this is: if it's a boolean, use "Is" prefix.

Reply:
yeah. I see.